pygments/lexers/ml.py

```diff
@@ -382,7 +382,7 @@ class OcamlLexer(RegexLexer):
     )
 
     operators = r'[!$%&*+\./:<=>?@^|~-]'
-    word_operators = ('and', 'asr', 'land', 'lor', 'lsl', 'lxor', 'mod', 'or')
+    word_operators = ('asr', 'land', 'lor', 'lsl', 'lxor', 'mod', 'or')
```

Comment:
Yes, they are different. `and` is a keyword and is used in several language constructs. `or` used to be an infix operator (and is still technically a reserved word) but is considered obsolete.

It would make some sense to also remove `or` from the list of infix operators.